test/unit/cli/build/build_hibf_chopper_test.cpp

```diff
@@ -37,39 +37,28 @@ TEST_F(build_hibf_chopper, pipeline)
 
     ASSERT_TRUE(std::filesystem::exists(data_filename.get_path()));
 
-    { // generate count file
-        char const * argv[] = {"./chopper-count",
-                               "--kmer-size",
-                               "19",
-                               "--disable-sketch-output",
-                               "--column-index",
-                               "2",
-                               "--threads",
-                               "1",
-                               "--input-file",
-                               data_filename.get_path().c_str(),
-                               "--output-prefix",
-                               count_prefix.c_str()};
-        int const argc = sizeof(argv) / sizeof(*argv);
-        seqan3::argument_parser parser{"chopper-count", argc, argv, seqan3::update_notifications::off};
-        chopper::count::execute(parser);
-    }
-
-    ASSERT_TRUE(std::filesystem::exists(count_prefix.string() + ".count"));
+    { // build layout
+        cli_test_result const result = execute_app("raptor",
+                                                   "layout",
+                                                   "--kmer-size",
+                                                   "19",
+                                                   "--disable-sketch-output",
+                                                   "--column-index",
+                                                   "2",
+                                                   "--threads",
+                                                   "1",
+                                                   "--input-file",
+                                                   data_filename.get_path().c_str(),
+                                                   "--tmax",
+                                                   "64",
+                                                   "--false-positive-rate",
+                                                   "0.05",
+                                                   "--output-filename",
+                                                   layout_filename.get_path().c_str());
```

Comment:
I personally prefer this, but only because I find it more readable:
```suggestion
        cli_test_result const result = execute_app("raptor layout",
                                                   "--kmer-size 19",
                                                   "--disable-sketch-output",
                                                   "--column-index 2",
                                                   "--threads 1",
                                                   "--input-file",
                                                   data_filename.get_path().c_str(),
                                                   "--tmax 64",
                                                   "--false-positive-rate 0.05",
                                                   "--output-filename",
                                                   layout_filename.get_path().c_str());
```